### PR TITLE
Logging docs updates according k8s v1.14 CRI changes

### PR DESCRIPTION
**Pull request by hisarbalik**

### CODE REVIEW

File: docs/logging/docs/01-01-logging.md [hunk 1]
```diff
@@ -5,3 +5,7 @@ title: Overview
 Logging in Kyma uses [Loki](https://github.com/grafana/loki) which is a Prometheus-like log management system. This lightweight solution, integrated with Grafana, is easy to understand and operate. Loki provides Promtail which is a log router for Docker containers. Promtail runs inside Docker, checks each container and routes the logs to the log management system.
 
 > **NOTE:** At the moment, Kyma provides an alpha version of the Logging component.
```
Comment (bszwarc) [inline docs/logging/docs/01-01-logging.md]:
```suggestion

```
I merged it with the next one to have one note and one caution.

File: docs/kyma/docs/04-03-cluster-installation.md [hunk 2]
```diff
@@ -171,10 +171,22 @@ Follow these steps:
     ```
     cat kyma-installer-cluster.yaml <(echo -e "\n---") kyma-config-cluster.yaml | sed -e "s/__DOMAIN__/$DOMAIN/g" | sed -e "s/__TLS_CERT__/$TLS_CERT/g" | sed -e "s/__TLS_KEY__/$TLS_KEY/g" | sed -e "s/__.*__//g" > my-kyma.yaml
     ```
-
+    
+    > **NOTE:** If you deploy Kyma with GKE version 1.12.6-gke.X and above, please use following steps to prepare the deployment file. 
+        
+    - Run this command if you use the `xip.io` default domain:
```
Comment (bszwarc) [inline docs/kyma/docs/04-03-cluster-installation.md]:
```suggestion
    - If you use the `xip.io` default domain, run:
```

File: docs/kyma/docs/04-03-cluster-installation.md [hunk 3]
```diff
@@ -171,10 +171,22 @@ Follow these steps:
...
+        
+    ```
+    cat kyma-installer-cluster.yaml <(echo -e "\n---") kyma-config-cluster.yaml | sed -e "s/__PROMTAIL_CONFIG_NAME__/promtail-k8s-1-14.yaml/g" | sed -e "s/__.*__//g" > my-kyma.yaml
+    ```
+    
+    - Run this command if you use your own domain:
```
Comment (bszwarc) [inline docs/kyma/docs/04-03-cluster-installation.md]:
```suggestion
    - If you use your own domain, run:
```

File: docs/kyma/docs/04-03-cluster-installation.md [hunk 4]
```diff
@@ -202,7 +214,18 @@ Follow these steps:
     ```
     (cat installation/resources/installer.yaml ; echo "---" ; cat installation/resources/installer-config-cluster.yaml.tpl ; echo "---" ; cat installation/resources/installer-cr-cluster.yaml.tpl) | sed -e "s/__DOMAIN__/$DOMAIN/g" |sed -e "s/__TLS_CERT__/$TLS_CERT/g" | sed -e "s/__TLS_KEY__/$TLS_KEY/g" | sed -e "s/__.*__//g" > my-kyma.yaml
     ```
+    > **NOTE:** If you deploy Kyma with GKE version 1.12.6-gke.X and above, please use following steps to prepare the deployment file. 
+        
+    - Run this command if you use the `xip.io` default domain:
```
Comment (bszwarc) [inline docs/kyma/docs/04-03-cluster-installation.md]:
```suggestion
    - If you use the `xip.io` default domain, run:
```

File: docs/kyma/docs/04-03-cluster-installation.md [hunk 5]
```diff
@@ -202,7 +214,18 @@ Follow these steps:
...
+    ```
+    (cat installation/resources/installer.yaml ; echo "---" ; cat installation/resources/installer-config-cluster.yaml.tpl ; echo "---" ; cat installation/resources/installer-cr-cluster.yaml.tpl) | sed -e "s/__PROMTAIL_CONFIG_NAME__/promtail-k8s-1-14.yaml/g" | sed -e "s/__.*__//g" > my-kyma.yaml
+    ```
 
+    - Run this command if you use your own domain:
```
Comment (bszwarc) [inline docs/kyma/docs/04-03-cluster-installation.md]:
```suggestion
    - If you use your own domain, run:
```

File: docs/kyma/docs/04-03-cluster-installation.md [hunk 6]
```diff
@@ -476,7 +499,19 @@ Follow these steps:
     ```
     cat kyma-installer-cluster.yaml <(echo -e "\n---") kyma-config-cluster.yaml | sed -e "s/__PROXY_EXCLUDE_IP_RANGES__/10.0.0.1/g" | sed -e "s/__DOMAIN__/$SUB_DOMAIN.$DNS_DOMAIN/g" | sed -e "s/__TLS_CERT__/$TLS_CERT/g" | sed -e "s/__TLS_KEY__/$TLS_KEY/g" | sed -e "s/__.*__//g" > my-kyma.yaml
     ```
+    
+    > **NOTE:** If you deploy Kyma with kubernetes version 1.14 and above, please use following steps to prepare the deployment file. 
+        
+    - Run this command if you use the `xip.io` default domain:
```
Comment (bszwarc) [inline docs/kyma/docs/04-03-cluster-installation.md]:
```suggestion
    - If you use the `xip.io` default domain, run:
```

File: docs/kyma/docs/04-03-cluster-installation.md [hunk 7]
```diff
@@ -476,7 +499,19 @@ Follow these steps:
...
+    ```
+    cat kyma-installer-cluster.yaml <(echo -e "\n---") kyma-config-cluster.yaml | sed -e "s/__PROMTAIL_CONFIG_NAME__/promtail-k8s-1-14.yaml/g" | sed -e "s/__PROXY_EXCLUDE_IP_RANGES__/10.0.0.1/g" | sed -e "s/__.*__//g" > my-kyma.yaml
+    ```
 
+    - Run this command if you use your own domain:
```
Comment (bszwarc) [inline docs/kyma/docs/04-03-cluster-installation.md]:
```suggestion
    - If you use your own domain, run:
```

File: docs/kyma/docs/04-03-cluster-installation.md [hunk 8]
```diff
@@ -507,7 +542,17 @@ Follow these steps:
     ```
     (cat installation/resources/installer.yaml ; echo "\n---" ; cat installation/resources/installer-config-cluster.yaml.tpl ; echo "\n---" ; cat installation/resources/installer-cr-cluster.yaml.tpl) | sed -e "s/__PROXY_EXCLUDE_IP_RANGES__/10.0.0.1/g" | sed -e "s/__DOMAIN__/$SUB_DOMAIN.$DNS_DOMAIN/g" | sed -e "s/__TLS_CERT__/$TLS_CERT/g" | sed -e "s/__TLS_KEY__/$TLS_KEY/g" | sed -e "s/__.*__//g" > my-kyma.yaml
     ```
+    > **NOTE:** If you deploy Kyma with kubernetes version 1.14 and above, please use following steps to prepare the deployment file. 
+    - Run this command if you use the `xip.io` default domain:
```
Comment (bszwarc) [inline docs/kyma/docs/04-03-cluster-installation.md]:
```suggestion
    - If you use the `xip.io` default domain, run:
```

File: docs/kyma/docs/04-03-cluster-installation.md [hunk 9]
```diff
@@ -507,7 +542,17 @@ Follow these steps:
...
+    ```
+    (cat installation/resources/installer.yaml ; echo "\n---" ; cat installation/resources/installer-config-cluster.yaml.tpl ; echo "\n---" ; cat installation/resources/installer-cr-cluster.yaml.tpl) | sed -e "s/__PROMTAIL_CONFIG_NAME__/promtail-k8s-1-14.yaml/g" | sed -e "s/__PROXY_EXCLUDE_IP_RANGES__/10.0.0.1/g" | sed -e "s/__.*__//g" > my-kyma.yaml
+    ```
 
+    - Run this command if you use your own domain:
```
Comment (bszwarc) [inline docs/kyma/docs/04-03-cluster-installation.md]:
```suggestion
    - If you use your own domain, run:
```

File: docs/kyma/docs/04-03-cluster-installation.md [hunk 10]
```diff
@@ -171,10 +171,22 @@ Follow these steps:
     ```
     cat kyma-installer-cluster.yaml <(echo -e "\n---") kyma-config-cluster.yaml | sed -e "s/__DOMAIN__/$DOMAIN/g" | sed -e "s/__TLS_CERT__/$TLS_CERT/g" | sed -e "s/__TLS_KEY__/$TLS_KEY/g" | sed -e "s/__.*__//g" > my-kyma.yaml
     ```
-
+    
+    > **NOTE:** If you deploy Kyma with GKE version 1.12.6-gke.X and above, follow these steps to prepare the deployment file. 
+        
+    - If you use the `xip.io` default domain, run:
```
Comment (bszwarc) [inline docs/kyma/docs/04-03-cluster-installation.md]:
```suggestion
    - Run this command if you use the xip.io default domain:
```

File: docs/kyma/docs/04-03-cluster-installation.md [hunk 11]
```diff
@@ -202,7 +214,18 @@ Follow these steps:
     ```
     (cat installation/resources/installer.yaml ; echo "---" ; cat installation/resources/installer-config-cluster.yaml.tpl ; echo "---" ; cat installation/resources/installer-cr-cluster.yaml.tpl) | sed -e "s/__DOMAIN__/$DOMAIN/g" |sed -e "s/__TLS_CERT__/$TLS_CERT/g" | sed -e "s/__TLS_KEY__/$TLS_KEY/g" | sed -e "s/__.*__//g" > my-kyma.yaml
     ```
+    > **NOTE:** If you deploy Kyma with GKE version 1.12.6-gke.X and above, follow these steps to prepare the deployment file. 
+        
+    - If you use the `xip.io` default domain, run:
```
Comment (bszwarc) [inline docs/kyma/docs/04-03-cluster-installation.md]:
```suggestion
    - Run this command if you use the xip.io default domain:
```

File: docs/kyma/docs/04-03-cluster-installation.md [hunk 12]
```diff
@@ -476,7 +499,19 @@ Follow these steps:
     ```
     cat kyma-installer-cluster.yaml <(echo -e "\n---") kyma-config-cluster.yaml | sed -e "s/__PROXY_EXCLUDE_IP_RANGES__/10.0.0.1/g" | sed -e "s/__DOMAIN__/$SUB_DOMAIN.$DNS_DOMAIN/g" | sed -e "s/__TLS_CERT__/$TLS_CERT/g" | sed -e "s/__TLS_KEY__/$TLS_KEY/g" | sed -e "s/__.*__//g" > my-kyma.yaml
     ```
+    
+    > **NOTE:** If you deploy Kyma with Kubernetes version 1.14 and above, follow these steps to prepare the deployment file. 
+        
+    - If you use the `xip.io` default domain, run:
```
Comment (bszwarc) [inline docs/kyma/docs/04-03-cluster-installation.md]:
```suggestion
    - Run this command if you use the xip.io default domain:
```

File: docs/kyma/docs/04-03-cluster-installation.md [hunk 13]
```diff
@@ -507,7 +542,17 @@ Follow these steps:
     ```
     (cat installation/resources/installer.yaml ; echo "\n---" ; cat installation/resources/installer-config-cluster.yaml.tpl ; echo "\n---" ; cat installation/resources/installer-cr-cluster.yaml.tpl) | sed -e "s/__PROXY_EXCLUDE_IP_RANGES__/10.0.0.1/g" | sed -e "s/__DOMAIN__/$SUB_DOMAIN.$DNS_DOMAIN/g" | sed -e "s/__TLS_CERT__/$TLS_CERT/g" | sed -e "s/__TLS_KEY__/$TLS_KEY/g" | sed -e "s/__.*__//g" > my-kyma.yaml
     ```
+    > **NOTE:** If you deploy Kyma with Kubernetes version 1.14 and above, follow these steps to prepare the deployment file. 
+    - If you use the `xip.io` default domain, run:
```
Comment (bszwarc) [inline docs/kyma/docs/04-03-cluster-installation.md]:
```suggestion
    - Run this command if you use the xip.io default domain:
```

File: docs/kyma/docs/04-03-cluster-installation.md [hunk 14]
```diff
@@ -507,7 +542,17 @@ Follow these steps:
...
+    ```
+    (cat installation/resources/installer.yaml ; echo "\n---" ; cat installation/resources/installer-config-cluster.yaml.tpl ; echo "\n---" ; cat installation/resources/installer-cr-cluster.yaml.tpl) | sed -e "s/__PROMTAIL_CONFIG_NAME__/promtail-k8s-1-14.yaml/g" | sed -e "s/__PROXY_EXCLUDE_IP_RANGES__/10.0.0.1/g" | sed -e "s/__.*__//g" > my-kyma.yaml
+    ```
 
+    - If you use your own domain, run:
```
Comment (bszwarc) [inline docs/kyma/docs/04-03-cluster-installation.md]:
```suggestion
    - Run this command if you use your own domain:
```

File: docs/kyma/docs/04-03-cluster-installation.md [hunk 15]
```diff
@@ -476,7 +499,19 @@ Follow these steps:
...
     ```
+    
+    > **NOTE:** If you deploy Kyma with Kubernetes version 1.14 and above, follow these steps to prepare the deployment file. 
+        
+    - If you use the `xip.io` default domain, run:
+    ```
+    cat kyma-installer-cluster.yaml <(echo -e "\n---") kyma-config-cluster.yaml | sed -e "s/__PROMTAIL_CONFIG_NAME__/promtail-k8s-1-14.yaml/g" | sed -e "s/__PROXY_EXCLUDE_IP_RANGES__/10.0.0.1/g" | sed -e "s/__.*__//g" > my-kyma.yaml
+    ```
 
+    - If you use your own domain, run:
```
Comment (bszwarc) [inline docs/kyma/docs/04-03-cluster-installation.md]:
```suggestion
    - Run this command if you use your own domain:
```

File: docs/kyma/docs/04-03-cluster-installation.md [hunk 16]
```diff
@@ -202,7 +214,18 @@ Follow these steps:
...
     (cat installation/resources/installer.yaml ; echo "---" ; cat installation/resources/installer-config-cluster.yaml.tpl ; echo "---" ; cat installation/resources/installer-cr-cluster.yaml.tpl) | sed -e "s/__DOMAIN__/$DOMAIN/g" |sed -e "s/__TLS_CERT__/$TLS_CERT/g" | sed -e "s/__TLS_KEY__/$TLS_KEY/g" | sed -e "s/__.*__//g" > my-kyma.yaml
     ```
+    > **NOTE:** If you deploy Kyma with GKE version 1.12.6-gke.X and above, follow these steps to prepare the deployment file. 
+        
+    - If you use the `xip.io` default domain, run:
+    ```
+    (cat installation/resources/installer.yaml ; echo "---" ; cat installation/resources/installer-config-cluster.yaml.tpl ; echo "---" ; cat installation/resources/installer-cr-cluster.yaml.tpl) | sed -e "s/__PROMTAIL_CONFIG_NAME__/promtail-k8s-1-14.yaml/g" | sed -e "s/__.*__//g" > my-kyma.yaml
+    ```
 
+    - If you use your own domain, run:
```
Comment (bszwarc) [inline docs/kyma/docs/04-03-cluster-installation.md]:
```suggestion
    - Run this command if you use your own domain:
```

File: docs/kyma/docs/04-03-cluster-installation.md [hunk 17]
```diff
@@ -171,10 +171,22 @@ Follow these steps:
...
+    
+    > **NOTE:** If you deploy Kyma with GKE version 1.12.6-gke.X and above, follow these steps to prepare the deployment file. 
+        
+    - If you use the `xip.io` default domain, run:
+        
+    ```
+    cat kyma-installer-cluster.yaml <(echo -e "\n---") kyma-config-cluster.yaml | sed -e "s/__PROMTAIL_CONFIG_NAME__/promtail-k8s-1-14.yaml/g" | sed -e "s/__.*__//g" > my-kyma.yaml
+    ```
+    
+    - If you use your own domain, run:
```
Comment (bszwarc) [inline docs/kyma/docs/04-03-cluster-installation.md]:
```suggestion
    - Run this command if you use your own domain:
```

**Comment by hisarbalik** [comment]
#3652

**Pull request merged**